models/message/message.js

```diff
@@ -38,6 +38,20 @@ class Message {
         return this._model.timestamp;
     }
 
+    get edits() {
+        return this._model.updates.map((edit) => ({
```

Comment:
Does this need to be wrapped in a `(`? Could you just do
```
return this._model.updates.map((edit) => {
   ...
});
```

Reply:
that does need the `(` because the `{}` is an object that is being returned not a block of code